History.md

```diff
@@ -1,5 +1,18 @@
 ## v.NEXT
 
+### Breaking changes
+N/A
+
+### Migration Steps
+N/A
+
+### Changes
+
+* Add `--local-server-port` option to override local port where Cordova will 
```

Comment:
What do you think about calling the option `--cordova-server-port` rather than `--local-server-port`, since it's Cordova that runs the local webapp server?

Reply:
Thank you for the review @benjamn 😄 

I agree, makes more sense. I already changed to `--cordova-server-port` and also updated the guide PR https://github.com/meteor/guide/pull/982

I hope everything is fine now.